source/includes/config-repo/_022-pipeline-export.md.erb

```diff
@@ -0,0 +1,132 @@
+## Pipeline export
+
+This message is a request to the plugin to convert a pipeline configuration into the plugin's native language. It is
+essentially the reverse of the [`parse-directory` request](#parse-directory).
+
+<p class='request-name-heading'>Request name</p>
+
+`pipeline-export`
+
+<p class='request-body-heading'>Request body</p>
+
+> Given the following config XML snippet (assume this pipeline is in a group named `my-group`) —
+
+```xml
+<pipeline name="my-pipeline">
+  <materials>
+    <git url="https://my.repo/path/to.git" />
+  </materials>
+  <stage name="first-stage">
+    <jobs>
+      <job name="first-job">
+        <tasks>
+          <exec command="/bin/echo">
+            <arg>hello</arg>
+            <runif status="passed" />
+          </exec>
+        </tasks>
+      </job>
+    </jobs>
+  </stage>
+</pipeline>
+```
+
+The request body will contain a JSON object with an attribute `pipeline`, which contains a JSON representation of a full pipeline configuration.
```

Comment:
You can mention the parse directory message too, if you want. It has a reasonably comprehensive example of all possible entities.

Reply:
Good point. I'll add that.